panpipes/panpipes/pipeline_preprocess/pipeline.yml

```diff
@@ -1,328 +1,203 @@
 # ============================================================
 # Preprocess workflow Panpipes (pipeline_preprocess.py)
 # ============================================================
-# written by Charlotte Rich-Griffin and Fabiola Curion
+# This file contains the parameters for the ingest workflow.
+# For full descriptions of the parameters, see the documentation at https://panpipes-pipelines.readthedocs.io/en/latest/yaml_docs/pipeline_preprocess_yml.html
 
 
-# ------------------------
-# compute resource options
-# ------------------------
+#--------------------------
+# Compute resources options
+#--------------------------
 resources:
-  # Number of threads used for parallel jobs
-  # this must be enough memory to load your mudata and do computationally intensive tasks
   threads_high: 2
-  # this must be enough memory to load your mudata and do computationally light tasks
   threads_medium: 2
-  # this must be enough memory to load text files and do plotting, requires much less memory than the other two
   threads_low: 1
-# path to conda env, leave blank if running native or your cluster automatically inherits the login node environment
+
 condaenv:
 
 # allows for tweaking where the jobs get submitted, in case there is a special queue for long jobs or you have access to a gpu
```

Comment:
What does this comment refer to? The `condaenv`? If yes, I don't think I understand what is meant by that.

Reply:
yes this refers to the path to the conda environment if you're running this workflow with a specific  conda env. 

notice you can install panpipes in a main conda env  (/path/to/panpipes_env/) and have a separate conda environment JUST for this workflow (path/to/preprocess_env).

running panpipes after activating the conda environment /path/to/panpipes_env/ should ensure that the environment is inherited when running the pipeline if you are on an interactive session (or on a PC) 

when jobs are spawn, we found that different clusters behave differently so we always leave the option to specify the conda env (whether it's /path/to/panpipes_env/ or /path/to/preprocess_env/) in the yml

there is a section about this in the installation, but maybe not superclear?
https://panpipes-pipelines.readthedocs.io/en/latest/install.html#specifying-conda-environments-to-run-panpipes

Reply:
Ah, I see - thank you for the explanation! I deleted that part from the `pipeline.yml` file itself but added the link you send here to the parameter documentation. I think that should clarify things.